app/Http/Controllers/LanguageController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Support\Facades\Session;
+
+use Illuminate\Http\Request;
+
+class LanguageController extends Controller
+{
+    public function changeLanguage(Request $request, $language)
+    {
+        Session::put('lang', $language);
+        return redirect()->back();
```

Comment:
```suggestion

        return redirect()->back();
```
Thêm dòng trống trc return e nhé